app/Http/Resources/UserResource.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Otoszroto\Http\Resources;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class UserResource extends JsonResource
+{
+    /**
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return [
+            "id" => $this->id,
+            "firstname" => $this->firstname,
+            "surname" => $this->surname,
+            "phone" => $this->phone,
+            "email" => $this->email,
+            "emailVerifiedAt" => $this->email_verified_at,
+            "createdAt" => $this->created_at,
+            "updatedAt" => $this->updated_at,
```

Comment:
```suggestion
            "phone" => $this->phone,
            "email" => $this->email,
            "createdAt" => $this->created_at,
            "updatedAt" => $this->updated_at,
```

`emailVerifiedAt` przydatny jest tylko na backendzie, nie ma sensu go przesyłać na front.

w realnej aplikacji `phone` i `email` powinny iść innym zapytaniem, żeby nie ułatwiać życia scrapperom. Nikt nie chciałby chyba mieć wycieku numeru telefonu i maila, bo zostawił komentarz pod produktem. Więc technicznie powinniśmy wtedy mieć kilka UserResource w zależności od kontekstu użycia. W naszej aplikacji to odpuszczamy, to tylko projekt studencki.